addons/common/fnc_execNextFrame.sqf

```diff
@@ -21,9 +22,17 @@ Author:
     esteldunedain and PabstMirror, donated from ACE3
 ---------------------------------------------------------------------------- */
 
-params [["_function", {}, [{}]], ["_args", []]];
+params [["_function", {}, [{}]], ["_args", []], ["_frames", 1, [0]]];
 
-if (diag_frameno != GVAR(nextFrameNo)) then {
+// Do not allow negative or 0 frame delays
+_frames = _frames max 1;
+
+if (_frames > 1) exitWith {
+    GVAR(nextFrameNBuffer) pushBack [diag_frameNo + _frames, _args, _function];
```

Comment:
I don't think this will work with save games
`diag_frameNo` will be completely different on load after restart

Reply:
Could add something like [this](https://github.com/CBATeam/CBA_A3/blob/c472830703a5c335bfa990c393a6b095d66d74bc/addons/common/init_perFrameHandler.sqf#L90) to fix that.